src/test/resources/roles_bulk.yml

```diff
@@ -17,6 +17,6 @@ bulk:
     allowed_actions:
     - "indices:data/write/index"
     - "indices:admin/mapping/put"
-    - "indices:admin/create"
+    - "indices:admin/auto_create"
```

Comment:
Should you have here both ? "create" and "auto_create" ?

Reply:
bulk requires only "auto_create". I can add "create", but it won't be used.

Reply:
Looking at the ES PR, I see that they added "auto_create" everywhere where "create" was present, so ideally we should have both, just in case. 

In addition to these changes, could you review all the places were we use "indices:admin/create" ?
Example:
https://github.com/opendistro-for-elasticsearch/security/blob/cbcc6245bd923c545fc7e52dabe1bed267ddbf6c/src/main/resources/static_config/static_roles.yml

https://github.com/opendistro-for-elasticsearch/security/blob/a3d7a56e7c21ff011dad66a75389d157732f43fc/src/main/resources/static_config/static_action_groups.yml

https://github.com/opendistro-for-elasticsearch/security/blob/3aef3a3143f30fb95ac499742b80e2de75a355d1/src/main/java/com/amazon/opendistroforelasticsearch/security/support/ConfigConstants.java

Reply:
We can follow the same approach and replace "auto_create" with "create" while checking if user is allowed to perform "auto_create" action.

Based on testing restoring snapshot does not require permission for "auto_create".